views/patterns/02-organisms/sections/page-header-hero.blade.php

```diff
@@ -119,6 +119,7 @@
           $image_l = wp_get_attachment_image_src($thumb_id, $thumb_size . '--l')[0];
           $image_xl = wp_get_attachment_image_src($thumb_id, $thumb_size . '--xl')[0];
           $alt = get_post_meta($thumb_id, '_wp_attachment_image_alt', true);
+          $title_h1 = true;
```

Comment:
@designerbrent This is the code that changes all the Hero titles to H1's instead of H3's. All the page headers use the `media-block`, so that files will conitionally check for `$title_h1` and switch the H tag.